### DISCUSSION TITLE
ControlNet for "Inpaint" clothing (or objects)

### DISCUSSION THREAD

**Discussion by trendland** [lllyasviel/ControlNet#440] [Ideas]
Been trying to crack this for months. I would've thought the new CN Inpaint would resolved that but nope. 
Basically, Im trying to have an image or a gen and inpainting a specific piece of clothing ( or object) into the image. 

Because now, with CN we can control pretty anything we want (lighting, positions, references, mood...) but not yet clothing! 

I've tried pretty much every CN stacked on top of each others and every settings but im not getting the results I want, so wondering if anyone found a way (instead of creating loras and inpainting) 
or maybe @lllyasviel has something up his sleeves coming up soon :) 

See my attempts screens below. 

<img width="477" alt="f74bdf77bbea0dbc2dac54b45c9f3ee4" src="https://github.com/lllyasviel/ControlNet/assets/7536142/a10a27a4-5236-43df-8082-56375d6a7678">

<img width="654" alt="2b0f4be967092bbff8860045a42b9864" src="https://github.com/lllyasviel/ControlNet/assets/7536142/3ceed9c9-17ea-47dd-8be9-c562e43d9c48">

<img width="285" alt="a7db8bfb467d8625a63779cd4331152e" src="https://github.com/lllyasviel/ControlNet/assets/7536142/787e14a2-b19d-441e-b703-b6d2b9885ea5">

**Comment by daffatahtaa** [comment]
Hello, have you found a working workflow?

**Reply by geroldmeisinger** [reply]
look here [Composer: Creative and Controllable Image Synthesis with Composable Conditions](https://arxiv.org/abs/2302.09778)
